### PR TITLE
Refactors picture frames to use wall frames system

### PR DESCRIPTION
**Pull request by Core0verload**
Why they are not wall frames to begin with is beyond me.

Other changes:
* Removed `apc_repair.dmi `, wall frame sprites are moved to `wallframe.dmi `.
* Photo icon is now centered properly.
* Picture frames are now a bit more compact and less ugly:
default
(new sprite is on the bottom)

### CODE REVIEW

File: code/modules/mob/living/carbon/human/human_defense.dm [hunk 1]
```diff
@@ -682,10 +682,10 @@
 					status += "numb"
 				if(status == "")
 					status = "OK"
-				to_chat(src, "\t [status == "OK" ? "\blue" : "\red"] Your [LB.name] is [status].")
+				to_chat(src, " [status == "OK" ? "\blue" : "\red"] Your [LB.name] is [status].")
```
Comment (optimumtact) [inline code/modules/mob/living/carbon/human/human_defense.dm]: why did you touch this code? it's entirely unrelated

File: code/modules/paperwork/photography.dm [hunk 2]
```diff
@@ -558,35 +553,33 @@
 	icon_state = "frame-empty"
 	var/obj/item/weapon/photo/framed
 
+/obj/structure/sign/picture_frame/New(loc, dir, building)
+	..()
+	if(dir)
+		src.setDir(dir)
```
Comment (optimumtact) [inline code/modules/paperwork/photography.dm]: you can just do setDir(dir) I believe

**Review by optimumtact — APPROVED** [review]
Some minor nits but okay to merge after they're addressed

**Comment by Core0verload** [comment]
"Minor nits" are addressed, ready to merge.

**Pull request merged**